### PR TITLE
Dinka Keyboard

### PR DESCRIPTION
**Pull request by MakaraSok**

### CODE REVIEW

File: el_dinka/source/el_dinka.kps [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  </Info>
+  <Files>
+    <File>
+      <Name>welcome.htm</Name>
+      <Description>File welcome.htm</Description>
+      <CopyLocation>0</CopyLocation>
+      <FileType>.htm</FileType>
+    </File>
+    <File>
+      <Name>el_dinka.kvk</Name>
```
Comment (mcdurdin) [inline el_dinka/source/el_dinka.kps]: This should be from ..\build\

Comment (mcdurdin) [inline el_dinka/source/el_dinka.kps]: The source file will eventually be a .kvks; the build version is guaranteed to have the correct metadata to link to the compiled keyboard

File: el_dinka/README.md [hunk 2]
```diff
@@ -11,10 +11,10 @@ Dinka Unicode keyboard layout for the Dinka Language Institute, Australia.
...
 -----
 
- * Home:     <___URL___>
- * Deployed: <___URL___>
- * Help:     <___URL___>
- * Contact:  <___EMAIL___>
+ * Home:     https://keyman.com
+ * Deployed: https://s.keyman.com/keyboard/dlia25bas
+ * Help:     https://help.keyman.com/keyboard/dlia25bas
+ * Contact:  support@keyman.com
```
Comment (mcdurdin) [inline el_dinka/README.md]: Contact should be lang.support@gmail.com

File: el_dinka/source/el_dinka.kps [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+  <Registry/>
+  <StartMenu>
+    <Folder></Folder>
+    <Items/>
+  </StartMenu>
+  <Info>
+    <Name URL="">el_dinka</Name>
+    <Version URL="">2.6</Version>
+    <Copyright URL="">© 2005-2017 State Library of Victoria and the Dinka Language Institute of Australia</Copyright>
+    <Author URL="">Andrew Cunningham</Author>
```
Comment (mcdurdin) [inline el_dinka/source/el_dinka.kps]: Could put contact email address in here as well, but not required.

**Comment by DavidLRowe** [comment]
- [ ] README.md
* In the line `Deployed: https://s.keyman.com/keyboard/dlia25bas`, the link says I'm not authorised to see the page. Perhaps better to remove that line.
* 'Contact:' change to 'lang.support@gmail.com'
* 'Todo:' I think the one item there can be removed (since all the blanks are filled in)

- [ ] welcome.htm is empty. I think this generates an empty window with an OK button, when installing the keyboard. @mcdurdin Is it better to have an empty file or no file at all?

- [ ] el_dinka.kps
README.md, HISTORY.md and LICENSE.md are included in the el_dinka.kps file, but I'm not sure that is correct. I don't think they are needed. 

- [ ] el_dinka.kmn
* Reference to &BITMAP "dlia2.bmp" is commented out. Is there an icon for the keyboard? If so, it should probably be named el_dinka.bmp for consistency.
* The Target line is commented out: `c store(&TARGETS) 'any windows'` It seems it should be 'windows' or 'windows macosx' (or maybe if it's absent that implies 'windows' or more likely 'windows macosx'?)

**Comment by DavidLRowe** [comment]
- [ ] README.md
* In the line `Deployed: https://s.keyman.com/keyboard/dlia25bas`, the link says I'm not authorised to see the page. Perhaps better to remove that line.
* 'Contact:' change to 'lang.support@gmail.com'
* 'Todo:' I think the one item there can be removed (since all the blanks are filled in)

- [ ] welcome.htm is empty. I think this generates an empty window with an OK button, when installing the keyboard. @mcdurdin Is it better to have an empty file or no file at all?

- [ ] el_dinka.kps
README.md, HISTORY.md and LICENSE.md are included in the el_dinka.kps file, but I'm not sure that is correct. I don't think they are needed. 

- [ ] el_dinka.kmn
* Reference to &BITMAP "dlia2.bmp" is commented out. Is there an icon for the keyboard? If so, it should probably be named el_dinka.bmp for consistency.
* The Target line is commented out: `c store(&TARGETS) 'any windows'` It seems it should be 'windows' or 'windows macosx' (or maybe if it's absent that implies 'windows' or more likely 'windows macosx'?)

**Comment by mcdurdin** [comment]
Notes on @DavidLRowe's comments:
* Deployed: let's go ahead and remove this line from all future keyboard README.md files. It doesn't belong any more given the current and future deployment strategies for Keyman keyboards.
* welcome.htm: It is better to remove this from the repo if it is empty. Better yet is to have some documentation; in fact I don't think we should publish it without any documentation.
* el_dinka.kps Correct re *.md files - don't include them.
* el_dinka.kmn: if the `&targets` store is missing or commented out, then that implies 'windows' (and 'macosx' as well.

**Comment by MakaraSok** [comment]
@DavidLRowe , I have checked and seen that most things have been updated according to your comments, but there are minor changes I made to the README.md. Please see the diff.

**Comment by DavidLRowe** [comment]
There's a note about .gitignore being a conflicting file. Does it need to be in the el_dinka folder? If not, perhaps it could be deleted?

Apart from that conflict, everything looks fine.

**Comment by MakaraSok** [comment]
@DavidLRowe There was a conflict on the branch and it needs to be resolved. Because of this, your review was dismissed. Please review it again and see if you like how everything looks.

**Comment by MakaraSok** [comment]
@DavidLRowe I've updated the welcome.htm and the screenshots of the keyboard. They should look consistent now in terms of font size.

**Comment by mcdurdin** [comment]
I've merged this but of course now see that dinkaweb11 is listed as a related keyboard but not dlia25bas. Given this keyboard is desktop only at present, it seems it shouldn't be deprecating dinkaweb11 but possibly should be deprecating dlia25bas?

**Comment by mcdurdin** [comment]
Keyboard currently published at: http://keymankeyboards.com/?id=314

**Pull request merged**